src/client/resources/conversions.ts

```diff
@@ -43,14 +43,22 @@ export function getTokenSecretLabelSelector(
   };
 }
 
-export function updateTokenSecret(rawToken: string) {
+export function updateTokenSecret(rawToken: string, isMigTokenSecret?: boolean) {
   // btoa => to base64, atob => from base64
   const encodedToken = btoa(rawToken);
-  return {
-    data: {
-      saToken: encodedToken,
-    },
-  };
+  if (isMigTokenSecret) {
```

Comment:
What's this `isMigTokenSecret` flag for? I noticed it's hard-coded to `true` above for now